libs/shared/src/types/protocol.ts

```diff
@@ -83,6 +83,8 @@ export enum WalletSsoSource {
   Twitter = 'twitter',
   Apple = 'apple',
   Email = 'email',
+  Farcaster = 'farcaster',
```

Comment:
is farcaster logic a part of this pr?

Reply:
@mzparacha this branch was made off of a branch Kurtis made, at the request of Dillon. So I believe that is a change he made.